### PR TITLE
Tenant Sign up and Login

### PR DESCRIPTION
**Pull request by fabianlundgren**
**PT Story:** https://www.pivotaltracker.com/story/show/144025203

Changes proposed in this PR:
- Adds sign up/login for tenants. 
- Adds a sign out tab.
- Basic design for sign up/login page. 
- Makes the sign up and login page root page

What I have learned working on this feature:
- To work with ng token auth and devise token auth. 
- To work with register and login forms in ionic. 

Screenshot:
skarmavbild 2017-04-25 kl 22 48 08

### CODE REVIEW

File: www/js/controllers.js [hunk 1]
```diff
@@ -1,25 +1,63 @@
...
+        .then(function(resp) {
+          $state.go('tab.home');
+        })
+        .catch(function(error) {
+          $scope.errors = error.errors;
+        });
+    }
+
+    $scope.toggleToLgnBtnClick = function() {
+      $scope.register = $scope.register !== true;
```
Comment (AmberWilkie) [inline www/js/controllers.js:51]: What does this do?

**Comment by AmberWilkie** [comment]
Can you guys actually see this working? We don't have any tests here, so I'm trusting that you are watching console logs and seeing that registration come back and / or watching the heroku logs to make sure that a user is actually being created or logged in.

File: www/js/controllers.js (hunk 1 above)
Comment (fottander) [reply www/js/controllers.js:51]: In the text already registered? sign in. If you toggle and click on sign in the form changes to  the sign in instead. And vice versa.

**Comment by fottander** [comment]
Yes it is working. We have tried it a lot in the browser.

**Comment by AmberWilkie** [comment]
Ok cool, I'll approve it.

**Review by AmberWilkie — APPROVED** [review]
Looks pretty good guys.

**Comment by AmberWilkie** [comment]
Have your colleagues press the button - after they review the code.

**Comment by Scharrre97** [comment]
I have try it as well and it works fantastic.

**Pull request merged**